content/influxdb/v2.0/reference/flux/stdlib/testing/loadstorage.md

```diff
@@ -0,0 +1,50 @@
+---
+title: testing.loadStorage() function
+description: The `testing.loadStorage()` function loads test data as if it were queried from InfluxDB.
+menu:
+  influxdb_2_0_ref:
+    name: testing.loadStorage
+    parent: Testing
+weight: 301
+---
+
+The `testing.loadStorage()` function loads test data as if it were queried from InfluxDB.
```

Comment:
as if the data were queried from InfluxDB (vs somewhere else?)

Reply:
Correct. Data could be queried from a SQL source, but this function is used in both the Flux and InfluxDB test suites. We currently don't have test suites for SQL databases.